waltz-data/src/main/java/com/khartec/waltz/data/server_information/ServerInformationDao.java

```diff
@@ -126,6 +126,14 @@ public ServerInformation getByExternalId(String externalId) {
     }
 
 
+    public ServerInformation getByHostname(String hostname) {
+        return dsl.select(SERVER_INFORMATION.fields())
+                .from(SERVER_INFORMATION)
```

Comment:
Not a biggie, but when doing this you can do: `dsl.selectFrom(SERVER_INFORMATION)` instead and save a bit of code.   Of course you have to put it back in again if you subsequently do a join.....